cmd/createtree/main_test.go

```diff
@@ -195,6 +186,25 @@ func runTest(t *testing.T, tests []*testCase) {
 	}
 }
 
+// expectCalls returns the minimum number of times a function is expected to be called
+// given the return error for the function (err), and all previous errors in the function's
+// code path.
+func expectCalls(err error, prevErr ...error) int {
```

Comment:
another option might be to have this take a `*gomock.Call` rather than returning an `int`, then it could choose to use `.AnyTimes`, `.MinTimes`, or .`Times` as appropriate?

e.g.
```golang
call := server.MockTrillianMapServer.EXPECT().GetSignedMapRootByRevision(gomock.Any(),  gomock.Any()).Return(&trillian.GetSignedMapRootResponse{}, nil)
expectCalls(call, retErr error, prevErr ...error)
```
wdyt?

Reply:
That would be slightly more precise wouldn't it?